gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -406,6 +411,15 @@ private void addPolygonFeature() {
         mapContainerViewModel.getSelectedLayer().get(), polygonPoints);
   }
 
+  private void undoPolygonVertice() {
+    if (polygonPoints.size() == 0) {
```

Comment:
->`polygonVerticies.isEmpty()`

Reply:
changed from polygonVertices.size() to polygonVerticies.isEmpty()